be/src/vec/columns/column_dictionary.h

```diff
@@ -258,9 +258,9 @@ class ColumnDictionary final : public COWHelper<IColumn, ColumnDictionary<T>> {
 
     uint32_t get_hash_value(uint32_t idx) const { return _dict.get_hash_value(_codes[idx]); }
 
-    phmap::flat_hash_set<int32_t> find_codes(
-            const phmap::flat_hash_set<StringValue>& values) const {
-        return _dict.find_codes(values);
+    void find_codes(const phmap::flat_hash_set<StringValue>& values,
+                    std::vector<bool>& selected) const {
+        return _dict.find_codes(values, selected);
```

Comment:
`find_codes` passes the result `selected` by reference, and not by `return`. This is not consistent with other interfaces, such as `find_code`, `find_code_by_bound`. I think it's better to keep it uniform.